### PR TITLE
appveyor openblas error (taken over Added Appveyor CI #10)

### PR DESCRIPTION
**Pull request by Soonmok**

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,125 @@
+clone_depth: 10
+
+environment:
+
+  BOOST_PROG_OPTION: "C:/projects/mlpack/\
```
Comment (zoq) [inline .appveyor.yml]: You have to change the project path from ``C:/projects/mlpack/`` to ``C:/projects/ensmallen/``

Comment (Soonmok) [reply .appveyor.yml]: Thanks ! i will edit it !

File: .appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  BOOST_SERIALIZATION: "C:/projects/ensmallen/\
+  boost_serialization-vc140.1.60.0.0/lib/native/address-model-64/lib/*.*"
+  BOOST_UNIT_TEST: "C:/projects/ensmallen/\
+  boost_unit_test_framework-vc140.1.60.0.0/lib/native/address-model-64/lib/*.*"
+  ARMADILLO_INCLUDE: "C:/projects/ensmallen/armadillo-7.800.2/include"
+  ARMADILLO_LIBRARY: "C:/projects/ensmallen/armadillo-7.800.2/\
+  build/Debug/armadillo.lib"
+  BLAS_LIBRARY: "C:/projects/ensmallen/OpenBLAS.0.2.14.1/lib/native/lib/x64/\
+  libopenblas.dll.a"
+  MLPACK_DIR: "C:/projects/mlpack"
```
Comment (zoq) [inline .appveyor.yml]: Either you change this path as well, or you create the directory.

**Comment by Soonmok** [comment]
How can I test appveyor in local? 
It is really time-consuming to wait the result...

**Comment by zoq** [comment]
Sorry for the slow response, the build is based on the visual studio, see http://www.mlpack.org/doc/mlpack-3.0.4/doxygen/build_windows.html for build instructions.

**Pull request closed without merging**